app/presenters/publishing_api_presenters/links_presenter.rb

```diff
@@ -42,6 +46,33 @@ def policy_area_ids
       (item.try(:topics) || []).map(&:content_id)
     end
 
+    def related_guides_ids
+      item.try(:related_detailed_guide_ids) || []
+    end
+
+    def related_mainstream_ids
```

Comment:
There's an endpoint on publishing-api to do this more efficiently:

http://www.rubydoc.info/gems/gds-api-adapters/GdsApi/PublishingApiV2#lookup_content_ids-instance_method

As a rule, publishing-apps should only talk to publishing-api and frontend apps should only talk to content-store.